modules/social_features/social_follow_user/social_follow_user.install

```diff
@@ -149,3 +156,28 @@ function social_follow_user_update_11203(): string {
   // Output logged messages to related channel of update execution.
   return $update_helper->logger()->output();
 }
+
+/**
+ * Change a weigh of the social_follow_user module.
+ */
+function social_follow_user_update_11401(): void {
+  module_set_weight('social_follow_user', 6);
+}
+
+/**
+ * Enable following status of all existing users by default.
+ */
+function social_follow_user_update_11402(): void {
```

Comment:
@volodymyr-sydor It would be good to avoid this update hook, since ability to follow should be enabled by default for all users. Also on platforms where there are 50k of users it may such update may out of memory, so we would need to do it with batch.